scripts/setup/create_singlenode_container.sh

```diff
@@ -65,12 +67,15 @@ server_exec() {
     # server_exec 'sleep 5'
 
     # Get loader and dependencies.
-    server_exec "git clone --branch=$LOADER_BRANCH git@github.com:eth-easl/loader.git"
+    server_exec "git clone --branch=$LOADER_BRANCH https://github.com/wanghanchengchn/invitro.git loader"
     server_exec 'echo -en "\n\n" | sudo apt-get install python3-pip python-dev'
     server_exec 'cd; cd loader; pip install -r config/requirements.txt'
 
+    server_exec 'sudo wget -q https://github.com/mikefarah/yq/releases/download/v4.27.3/yq_linux_amd64 -O /usr/bin/yq && sudo chmod +x /usr/bin/yq'
+    server_exec '~/loader/scripts/setup/rewrite_yaml_files.sh'
```

Comment:
I think that's too late to do that, files are used during create_one_node_cluster. Maybe move the cloning loader to earlier point.